code/datums/supplypacks/science.dm

```diff
@@ -16,32 +16,13 @@
 	containertype = /obj/structure/largecrate
 	containername = "\improper coolant tank crate"
 
-/decl/hierarchy/supply_pack/science/mecha_ripley
-	name = "Circuit Crate (\"Ripley\" APLU)"
-	contains = list(/obj/item/weapon/book/manual/ripley_build_and_repair,
-					/obj/item/weapon/circuitboard/mecha/ripley/main, //TEMPORARY due to lack of circuitboard printer,
-					/obj/item/weapon/circuitboard/mecha/ripley/peripherals) //TEMPORARY due to lack of circuitboard printer
-	cost = 30
-	containertype = /obj/structure/closet/crate/secure
-	containername = "\improper APLU \"Ripley\" Circuit Crate"
-	access = access_sciencelvl2
-
-/decl/hierarchy/supply_pack/science/mecha_odysseus
-	name = "Circuit Crate (\"Odysseus\")"
-	contains = list(/obj/item/weapon/circuitboard/mecha/odysseus/peripherals, //TEMPORARY due to lack of circuitboard printer,
-					/obj/item/weapon/circuitboard/mecha/odysseus/main) //TEMPORARY due to lack of circuitboard printer
-	cost = 25
-	containertype = /obj/structure/closet/crate/secure
-	containername = "\improper \"Odysseus\" Circuit Crate"
-	access = access_sciencelvl2
-
 /decl/hierarchy/supply_pack/science/phoron
-	name = "Phoron assembly crate"
+	name = "Assembly crate"
 	contains = list(/obj/item/weapon/tank/phoron = 3,
```

Comment:
Let's make sure to rename phoron.

Reply:
Eh, just keep Phoron in. No use changing it half-ass right now. Since more un-lore stuff is present as far as that goes, like the actual gas. Also. What's Phoron used for, those tanks? Nothing much afaik. Bomb making, which we don't have. And flamethrowers? Don't know if those would work without atmos, since they burn up oxy.

Reply:
Fuel source for PACMAN generators, can be used in R&D to increase Phoron research, so on. Used in creation of certain objects in R&D, used in creation of mech attachments.

Reply:
This is a tank though, not a sheet. Isn't it that any PACMAN uses phoron sheets? Mech was also removed. Like I said, I think you might be thinking of sheets?